mongo/client.go

```diff
@@ -696,13 +719,13 @@ func WithSession(ctx context.Context, sess Session, fn func(SessionContext) erro
 	return fn(contextWithSession(ctx, sess))
 }
 
-// UseSession creates a default session, that is only valid for the
+// UseSession creates a new session that is only valid for the
 // lifetime of the closure. No cleanup outside of closing the session
```

Comment:
Nit: It's not necessarily a closure.  It could be any function.

Reply:
Good point. Changed to `callback`.